python/lsst/ts/xml/data/sal_interfaces/MTHexapod/MTHexapod_Events.xml

```diff
@@ -411,4 +411,28 @@
       <Count>1</Count>
     </item>
   </SALEvent>
+  <SALEvent>
+    <Subsystem>MTHexapod</Subsystem>
+    <EFDB_Topic>MTHexapod_logevent_highCurrent</EFDB_Topic>
+    <Description>Indicates whether or not that the actuators are experiencing the high current that might come from the unbalanced loading, the cooling greasing, or others when the hexapod stills.</Description>
+    <item>
+      <EFDB_Name>highCurrent</EFDB_Name>
+      <Description>True if the actuators experience the high current. Otherwise, False.</Description>
+      <IDL_Type>boolean</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+  </SALEvent>
+  <SALEvent>
+    <Subsystem>MTHexapod</Subsystem>
+    <EFDB_Topic>MTHexapod_logevent_transitionToIdle</EFDB_Topic>
```

Comment:
Isn't this information already available in the `MTHexapod_logevent_controllerState` event? Is there really a need to publish just this boolean flag?

Don't want to be picky but I think something like `driversIdle` for a topic name and `idle` for attribute would probably be better.